database/mongodb/mongodb_test.go

```diff
@@ -32,10 +32,10 @@ var (
 	opts = dktest.Options{PortRequired: true, ReadyFunc: isReady}
 	// Supported versions: https://www.mongodb.com/support-policy
 	specs = []dktesting.ContainerSpec{
-		{ImageName: "mongo:3.4", Options: opts},
```

Comment:
Confirming that mongo db v5-v8 is supported:
https://www.mongodb.com/legal/support-policy/lifecycles